<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistraVagaForm</class>
 <widget class="QDialog" name="RegistraVagaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salvar Vaga</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/vaga.png</normaloff>../../Imagens/vaga.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ImageVagalabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="DescricaoVagaEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="SalvarVagaButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
